tests/MongoDB.Driver.Core.TestHelpers/FailPoint.cs

```
@@ -167,7 +168,7 @@ public void Dispose()
         {
             if (!_disposed)
             {
-                try { ConfigureOff(); } catch { }
+                ConfigureOff();
```

in this case, it's better to throw an exception rather than silently lose failpoint exception

---

Agreed.